packages/flutter/test/material/material_test.dart

```diff
@@ -321,6 +321,26 @@ void main() {
     expect(pressed, isTrue);
   });
 
+  testWidgets('BlankMaterial correctly implements MaterialInkController', (WidgetTester tester) async {
+    Color? color;
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Scaffold(
+          body: BlankMaterial(
+            color: Colors.cyan,
+            child: Builder(
+              builder: (BuildContext context) {
+                color = Material.maybeOf(context)?.color;
+                return const SizedBox.expand();
+              },
+            ),
+          ),
+        ),
+      ),
+    );
+    expect(color, equals(Colors.cyan));
+  });
```

Comment:
If you don't count stuff in `flutter/test/`, this pull request is a net decrease in line length. 😎